operations/customers.md

```diff
@@ -57,8 +60,16 @@ Returns all customers filtered by identifiers, emails, names and other filters.
 | `LoyaltyCodes` | array of string | optional, max 1000 items | Loyalty codes of the [Customer](customers.md#customer)s. |
 | `CreatedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which [Customer](customers.md#customer) was created. |
 | `UpdatedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which [Customer](customers.md#customer) was updated. |
+| `DeletedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which [Customer](customers.md#customer) was deleted. |
+| `ActivityStates` | array of string [Activity state](#activity-state) | optional | Whether return only active, only deleted or both records. |
 | `Extent` | [Customer extent](customers.md#customer-extent) | required | Extent of data to be returned. |
 
+#### Activity state
+
+* `Active` - Active records.
+* `Deleted`- Deleted records.
+* `All`- Active and Deleted records.
+
```

Comment:
The activity states should be defined somewhere in documentation (albeit in different file), reuse that.

Reply:
Yeah it is, but `All` wasn't there and I wanted to use it as one of the possible options. I could modify the origin definition, hopefully not breaking other usages.